client/src/components/Nav/NavLinks.jsx

```diff
@@ -15,6 +15,7 @@ import { cn } from '~/utils/';
 import DotsIcon from '../svg/DotsIcon';
```

Comment:
Please make sure all SVGs are exported from svg/index.ts

Also, import from `~/components` not `~/components/svg`